tests/unit/drivers/vector/test_griptape_cloud_knowledge_base_vector_store_driver.py

```diff
@@ -0,0 +1,78 @@
+import uuid
+from unittest.mock import Mock, patch
+from griptape.drivers import GriptapeCloudKnowledgeBaseVectorStoreDriver
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code):
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self):
+        return self.json_data
+
+
+test_ids = [str(uuid.uuid4()), str(uuid.uuid4())]
+test_vecs = [[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]]
+test_namespaces = [str(uuid.uuid4()), str(uuid.uuid4())]
+test_metas = [{"key": "value1"}, {"key": "value2"}]
+test_scores = [0.7, 0.8]
+test_entries = {
+    "entries": [
+        {
+            "id": test_ids[0],
+            "vector": test_vecs[0],
+            "namespace": test_namespaces[0],
+            "meta": test_metas[0],
+            "score": test_scores[0],
+        },
+        {
+            "id": test_ids[1],
+            "vector": test_vecs[1],
+            "namespace": test_namespaces[1],
+            "meta": test_metas[1],
+            "score": test_scores[1],
+        },
+    ]
+}
+
+
+class TestGriptapeCloudKnowledgeBaseVectorStoreDriver:
+    def mock_requests_post(*args, **kwargs):
+        return MockResponse(test_entries, 200)
```

Comment:
Mock how we do it [here](https://github.com/griptape-ai/griptape/blob/c9da271797dd69655f6a9b31de5be2da6ac3f827/tests/unit/tools/test_griptape_cloud_knowledge_base_client.py#L7-L23).

I assume you borrowed this style from [here](https://github.com/griptape-ai/griptape/blob/c9da271797dd69655f6a9b31de5be2da6ac3f827/tests/unit/tools/test_openweather_client.py#L29) which should probably be updated at some point.